benchmark/plot.sh

```diff
@@ -0,0 +1,50 @@
+# pip install openrlbenchmark==0.2.1a5
```

Comment:
Maybe we can do this along with the full CI PR like https://github.com/vwxyzjn/test-ci/blob/main/benchmark/plot.sh https://github.com/vwxyzjn/test-ci/blob/main/benchmark.sh? The PR as it stands just adds documentation. I could not chain something like `make full_run` with `make plot` because it's unclear what benchmarks have been run.

Reply:
Actually just added the benchmark scripts and updated makefile :)